config/general.yml-example

```diff
@@ -1018,6 +1018,13 @@ RESTRICTED_COUNTRIES: ''
 # ---
 NEW_REQUEST_RECAPTCHA: false
 
+# Show a reCAPTCHA on the contact form to discourage spammers
+#
+# CONTACT_FORM_RECAPTCHA – Boolean (default: false)
+#
+# ---
+CONTACT_FORM_RECAPTCHA: false
```

Comment:
I think a config key is a good option, given you need to explicitly account for it in the theme.